### ISSUE TITLE
Multi-label classification with two labels 

### ISSUE THREAD

**Issue by qingquansong** [keras-team/autokeras#1120]
### Bug Description

ImageClassifier classification head treats the multi-label classification with 2 labels as multi-class classification one-hot encoded labels.

### Bug Reproduction
Code for reproducing the bug:

----

from sklearn.datasets import make_multilabel_classification
X, Y = make_multilabel_classification(n_samples=100, n_features = 64,
                                      n_classes=2, n_labels=1,
                                      allow_unlabeled=False,
                                      random_state=1)
X = X.reshape((100, 8, 8))

clf = ak.ImageClassifier(max_trials=2, multi_label=True)
clf.fit(X, Y, epochs=3, verbose=2)

----

Data used by the code:
synthetic data created with scikit-learn

### Setup Details
Include the details about the versions of:
 - OS type and version:
 - Python: 3.6
 - autokeras: 1.0.2
 - keras-tuner:
 - scikit-learn:
 - numpy:
 - pandas:
 - tensorflow: 2.1.0
